src/lib/d3-compat.js

```diff
@@ -1,37 +1,36 @@
-(function() {
-
+(function () {
```

Comment:
Should we drop the d3 license into this directory? https://github.com/d3/d3/blob/main/LICENSE

Reply:
These aren't from d3 to my knowledge, I think they are extensions to d3 written for geojson.io.  These predate our current work and aren't new, they are just moved to a different directory with this PR.